### PR TITLE
Expand tutorials documentation, encourage contributors

### PR DESCRIPTION
**Pull request by davetcoleman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,34 @@
 # MoveIt! Tutorials
 
-This repository is automatically built by two systems:
-* The Travis build output is immediately deployed to gh-pages here: https://ros-planning.github.io/moveit_tutorials/
-* The ROS build farm output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
+[https://ros-planning.github.io/moveit_tutorials/](https://ros-planning.github.io/moveit_tutorials/)
 
-The tutorials use the [reStructuredText](http://www.sphinx-doc.org/en/stable/rest.html) format commonly used in the Sphinx "Python Documentation Generator". This unfortunately differs from the common Markdown format.
+This is the primary documentation for the MoveIt! project. We strongly encourage you to help improve MoveIt!'s documentation. Please consider reading the guidelines below for writing the best documentation and tutorials. However, if you are uncomfortable with any of the approaches, simply adding documentation texts to your pull requests is better than nothing.
```
Comment (nbbrooks) [inline README.md]: texts -> text

File: README.md [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
 
-## Travis Continuous Integration
+These tutorials use the [reStructuredText](http://www.sphinx-doc.org/en/stable/rest.html) format commonly used in the Sphinx "Python Documentation Generator". This unfortunately differs from the common Markdown format, but its advantage is that it supports embedding code directly from source files for inline code tutorials.
 
-[![Build Status](https://travis-ci.org/ros-planning/moveit_tutorials.svg?branch=master)](https://travis-ci.org/ros-planning/moveit_tutorials)
+This repository is currently built automatically by two systems:
+- [![Travis Build Status](https://travis-ci.org/ros-planning/moveit_tutorials.svg?branch=master)](https://travis-ci.org/ros-planning/moveit_tutorials) [Github Pages + Travis](https://ros-planning.github.io/moveit_tutorials/): latest
+- [![ROS Kinetic Builfarm Status](http://build.ros.org/buildStatus/icon?job=Kdoc__moveit_tutorials__ubuntu_xenial_amd64)](http://build.ros.org/job/Kdoc__moveit_tutorials__ubuntu_xenial_amd64/) [ROS Kinetic Build Farm](http://docs.ros.org/kinetic/api/moveit_tutorials/html/)
```
Comment (nbbrooks) [inline README.md]: Builfarm -> Build Farm

File: README.md [hunk 3]
```diff
@@ -1,36 +1,34 @@
...
+- [![Build Status](http://build.ros.org/buildStatus/icon?job=Idoc__moveit_tutorials__ubuntu_trusty_amd64)](http://build.ros.org/job/Idoc__moveit_tutorials__ubuntu_trusty_amd64/) [ROS Indigo Build Farm](http://docs.ros.org/indigo/api/moveit_tutorials/html/)
```
Comment (nbbrooks) [inline README.md]: nit: Build Status -> ROS Indigo Build Farm Status

File: README.md (hunk 1 above)
Comment (davetcoleman) [reply README.md]: fixed in separate commit

File: README.md (hunk 2 above)
Comment (davetcoleman) [reply README.md]: fixed in separate commit

File: README.md (hunk 3 above)
Comment (davetcoleman) [reply README.md]: fixed in separate commit

**Pull request merged**